codewind-che-sidecar/tests/sidecarfvt.bats

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bats
+
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+# Load common utility functions
+load testutil
+
+# Setup code to run before each test
+setup() {
+    export time_before=$SECONDS
+
+    if [ -z "$CLUSTER_IP" ]; then
+        echo "# Cluster IP is not defined in " '$CLUSTER_IP' >&3
+        exit 1
+    fi
+
+    if [ -z "$CHE_NAMESPACE" ]; then
+        echo "# Che namespace is not defined in " '$CHE_NAMESPACE' >&3
+        exit 1
+    fi
+
+    export CODEWIND_DEVFILE_URL=https://raw.githubusercontent.com/eclipse/codewind-che-plugin/master/devfiles/0.2.0/devfile.yaml
+    export CODEWIND_DEVILE_JSON=devfile.json
+    export CHE_INGRESS_DOMAIN=http://che-$CHE_NAMESPACE.$CLUSTER_IP.nip.io
```

Comment:
Since the Che ingress domain may vary across platforms and installation methods, I recommend we change this so that the entire Che ingress domain is passed in (such as `che-eclipse-che.9.1.2.3.nip.io`) instead of just the cluster IP.